src/Components/Artwork/FlatGridItem.tsx

```diff
@@ -25,7 +23,7 @@ const FlatGridItem: React.FC<FlatGridItemProps> = ({ artwork, onClick }) => {
   const { user } = useSystemContext()
   const isTeam = userIsTeam(user)
   const { containerProps, isSaveButtonVisible } = useSaveButton({
-    isSaved: !!artwork.isSaved,
+    isSaved: false,
```

Comment:
This was an issue. `isSaved` was being requested _outside_ of the save button context, and right in this component. It was passed to the `useSaveButton` hook - which effectively sets `isHover: true` - thus letting the old-school deprecated button show up always if you've saved the work (even w/o hovering).

With this change, the 'saved' status doesn't show up unless you hover.

Seems relatively minor to me 🙏